posthog/api/test/dashboards/test_dashboard.py

```diff
@@ -940,3 +940,49 @@ def test_relations_on_insights_when_dashboards_were_deleted(self) -> None:
         dashboard_two_json = self.dashboard_api.get_dashboard(dashboard_two_id)
         expected_dashboards_on_insight = dashboard_two_json["tiles"][0]["insight"]["dashboards"]
         assert expected_dashboards_on_insight == [dashboard_two_id]
+
+    def test_create_from_template_json(self) -> None:
```

Comment:
we should have the `test_invalid_template_receives_400_response` test at some point

Reply:
(because otherwise it gets context free 500)